example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/TestAccountStore.java

```diff
@@ -33,24 +35,25 @@ public TestAccountStore(Context context) {
 //        mAccountManager = AccountManager.get(context);
     }
 
+    @Nullable
     @Override
     public VimeoAccount loadAccount() {
         return AccountPreferenceManager.getClientAccount();
     }
 
-    @Deprecated
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
+    public void saveNonUserAccount(@NonNull VimeoAccount vimeoAccount) {
```

Comment:
@alfiehanssen a user can generate an auth token via the web API console, and use with the networking library. I was under the assumption this is not a client credentials grant. Do you have a suggestion for a name that accounts (pun intended) for a client credentials grant and that auth type?